Update LS-out-ITU-reply to-1983.md (#1)
IETF-OPS-AD/Liaisons: LS-out-ITU-reply to-1983.md

does not aim to alter any SRv6 specification or seek to specify new SRv6 requirements outside of IETF's work.

The specifications listed as "Core SRv6 Standards" in the attached document to
the LS are a subset of SRv6-related Proposed Standards published so far by the IETF. The IETF does not
identify a subset of SRv6-related specifications as "Core SRv6 Standards". 

> At the time of this LS response, there are several Working Groups (WGs) in the IETF 

Reviewer: ```suggestion
There are several Working Groups (WGs) in the IETF 
```

Reviewer: It is clear this is at the time of writing.

> (mainly spring, 6man, bess, idr, lsr, pce, rtgwg, teas, and opsawg) that are working

Reviewer: ```suggestion
(mainly SPRING, 6MAN, BMWG, BESS, IDR, LSR, PCE, RTGWG, TEAS, and OPSAWG) that are working
```

Reviewer: Also, include bmwg

> on SRv6 related specifications. The links to all active IETF WGs can be found at [2].

Reviewer: ```suggestion
on SRv6-related specifications. The links to all active IETF WGs can be found at [2].
```

Reviewer: nit fixing

Further, the list of RFCs published by each of the WG are available on each WG's page.

Unlike what is stated in the report, some of the listed RFCs (e.g., RFC 8665, 8666, and 8667) do not
apply for SRv6 but for SR-MPLS. Some RFCs (e.g., RFC 9256) apply for both SRv6 and SR-MPLS.
A review of the RFCs listed on the WGs pages will provide information on their applicability to SRv6.
Also, the correct names for these RFCs are as follows:

> 

Reviewer: ```suggestion
   * RFC 8665: OSPF Extensions for Segment Routing
   * RFC 8666: OSPFv3 Extensions for Segment Routing
   * RFC 8667: IS-IS Extensions for Segment Routing
```

Reviewer: I prefer to give this concrete correction

> Note that some other SRv6-related specifications (notably, "Compressed SRv6 Segment

Reviewer: ```suggestion
```

> List Encoding (CSID) [3]) may be approved for publication as an RFC and are under

Reviewer: ```suggestion

```

> the publication process with the RFC Editor.

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion

Note that some other SRv6-related specifications are being finalized by SPRING.
Notably, "Compressed SRv6 Segment List Encoding (CSID)" [3] has been approved
and is planned for publication by the IETF in the coming few months.
```

Reviewer: I prefer the OLD version

When referring to aspects of the IETF standards as "mandatory" or "optional", it is recommended to 

> also provide a reference to the specific section and text from the specific RFC. This will

Reviewer: ```suggestion
provide a reference to the specific section and text from the specific RFC. This will
```

> avoid misalignment with IETF standards. E.g., RFC 8754 does not specify the usage of Tag in 

Reviewer: ```suggestion
avoid misalignment with IETF standards. For example, RFC 8754 does not specify the usage of "Tag" in 
```

> the SRH (it only introduces the Tag field in the SRH) but the document marks 

Reviewer: ```suggestion
the SRH (it only introduces the Tag field in the SRH) but the ITU document marks 
```

Reviewer: Make it clear this is about the ITU doc
